homeassistant/components/sensor/eliqonline.py

```diff
@@ -0,0 +1,86 @@
+"""
+homeassistant.components.sensor.eliqonline
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+monitors home energy use for the eliq online service
+
+api documentation:
+  https://my.eliq.se/knowledge/sv-SE/49-eliq-online/299-eliq-online-api
+
+access to api access token:
+  https://my.eliq.se/user/settings/api
+
+current energy use:
+  https://my.eliq.se/api/datanow?accesstoken=<token>
+
+history:
+  https://my.eliq.se/api/data?startdate=2015-12-14&intervaltype=6min&accesstoken=<token>
+
+"""
+
+import logging
+from datetime import timedelta
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+from homeassistant.const import (STATE_UNKNOWN, CONF_ACCESS_TOKEN, CONF_NAME)
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['eliqonline==1.0.11']
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=30)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """ Set up the sensors """
+
+    import eliqonline
+
+    access_token = config.get(CONF_ACCESS_TOKEN)
```

Comment:
If there is no `access_token` the configuration.yaml file then `config.get(CONF_ACCESS_TOKEN)` will return `None`.